src/components/TodoItem.js

```diff
@@ -3,16 +3,24 @@ import PropTypes from 'prop-types';
 
 class TodoItem extends PureComponent {
```

Comment:
Using Class components with PureComponents risks your component to risk **prop changes will be ignored If it couldn't find a difference in shallow comparison**  I am sure your using it for the right reason 😊 performance 🚀

Reply:
@isaacpitwa , thanks for the review 😎